roles/ceph-mon/tasks/docker/configure_ceph_command_aliases.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: configure ceph profile.d aliases
+  template:
+    src: ceph-aliases.sh.j2
+    dest: "/etc/profile.d/ceph-aliases.sh"
+    mode: 0755
+    owner: root
+    group: root
+  become: true
```

Comment:
Sorry, this slipped from my radar, I don't think this is needed since we are calling the play with `become: true` already. @guits what do you think?

Reply:
You might be right, on the other hand I got this other template task as an example, which happens to have become: true

https://github.com/jtudelag/ceph-ansible/blob/3b0c6dc0172f660b59e79bc4c4c90b8a42b6c7f9/roles/ceph-mon/tasks/docker/start_docker_monitor.yml#L44

Reply:
Hum this might need to be removed as well then, but not in this PR. Please remove it :).

Reply:
@leseb true, although it's not breaking things, it's not necessary to have this. I guess it's just a leftover that could be nice to clean indeed